docs/README-migration.md

```diff
@@ -162,8 +162,31 @@ M_PI is no longer defined in SDL_stdinc.h, you can use the new symbols SDL_PI_D
 
 ## SDL_surface.h
 
-Removed unused 'depth' parameter from SDL_CreateRGBSurfaceWithFormat() and SDL_CreateRGBSurfaceWithFormatFrom()
-Removed unused 'flags' parameter from SDL_CreateRGBSurface() and SDL_CreateRGBSurfaceWithFormat()
+Removed SDL_CreateRGBSurface() and SDL_CreateRGBSurfaceFrom()
+SDL_CreateRGBSurfaceWithFormat() is renamed to SDL_CreateRGBSurface()
+SDL_CreateRGBSurfaceWithFormatFrom() is renamed to SDL_CreateRGBSurfaceFrom()
+
+Removed unused 'depth' parameter from SDL_CreateRGBSurface() and SDL_CreateRGBSurfaceFrom()
+Removed unused 'flags' parameter from SDL_CreateRGBSurface() 
```

Comment:
Can you combine these into a single change note and application recommendation?

e.g.
SDL_CreateRGBSurface() and SDL_CreateRGBSurfaceFrom() now take a format instead of masks, and SDL_CreateRGBSurface() no longer takes a flags parameter. SDL_CreateRGBSurfaceWithFormat() and SDL_CreateRGBSurfaceWithFormatFrom() are redundant and have been removed.

This code:
```
    SDL_Surface *surface = SDL_CreateRGBSurface(0, width, height, 0, Rmask, Gmask, Bmask, Amask);
```
can be replaced with this:
```
    Uint32 format = SDL_MasksToPixelFormatEnum(0, Rmask, Gmask, Bmask, Amask);
    SDL_Surface *surface = SDL_CreateRGBSurface(width, height, format);
```
but in general, you probably have a format that you know you're using, possibly one of these: